msal/src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -59,8 +59,6 @@ public sealed class TokenCache
         internal const string NullPreferredUsernameDisplayLabel = "Missing from the token response";
         private const string MicrosoftLogin = "login.microsoftonline.com";
```

Comment:
>MicrosoftLogin [](start = 29, length = 14)

this is also in B2C authority. perhaps it should be a const on the Authority object instead?

Reply:
I'll look at that when we go to cleanup the caching.